plugins/broker-redis/src/tests/beforeCreateServerPackageJson.spec.ts

```diff
@@ -3,27 +3,28 @@ import { mock } from "jest-mock-extended";
 import { name } from "../../package.json";
 import RedisBrokerPlugin from "../index";
 
-
 describe("Testing beforeServerPackageJson hook", () => {
-    let plugin: RedisBrokerPlugin;
-    let context: DsgContext;
-    beforeEach(() => {
-        plugin = new RedisBrokerPlugin();
-        context = mock<DsgContext>({
-            pluginInstallations: [{ npm: name }]
-        });
+  let plugin: RedisBrokerPlugin;
+  let context: DsgContext;
+  beforeEach(() => {
+    plugin = new RedisBrokerPlugin();
+    context = mock<DsgContext>({
+      pluginInstallations: [{ npm: name }],
     });
-    it("should add the dependencies required to use Redis to the package.json file", () => {
-        const { updateProperties } = plugin.beforeCreateServerPackageJson(context, {
-            fileContent: "",
-            updateProperties: [{}]
-        })
-        expect(updateProperties).toStrictEqual([{
-            dependencies: {
-                "redis": "3.1.2",
-                "@nestjs/microservices": "8.2.3",
-                "rxjs": "^7.8.1"
-            }
-        }]);
+  });
+  it("should add the dependencies required to use Redis to the package.json file", () => {
+    const { updateProperties } = plugin.beforeCreateServerPackageJson(context, {
+      fileContent: "",
+      updateProperties: [{}],
     });
+    expect(updateProperties).toStrictEqual([
+      {
+        dependencies: {
+          redis: "3.1.2",
+          "@nestjs/microservices": "8.2.3",
```

Comment:
Don't we need to change it to v10?

Reply:
Those changes are here:
https://github.com/amplication/plugins/pull/221